service.py

```diff
@@ -13,6 +13,12 @@ def debug(self, msg):
 
     def warning(self, msg):
         print("ydl warning: {}".format(msg), file=stderr)
+
+        # 429 responses that come through as warnings fail to
+        # be caught by the ytdl service, so we must escalate them
+        # to errors
+        if '429' in msg or 'Too Many Request' in msg:
```

Comment:
This is probably `Too Many Requests`, but I guess `Too Many Request` will work

Reply:
True, was just copying the condition that the `vivi-service-ytdl` checks, which is: `error.message.includes('Too Many Request')`, will keep it the same for consistency between the two since they're checking for the same thing.

Reply:
Looks like this typo (?) has been there for a long time!